### DISCUSSION TITLE
Change scroll value

### DISCUSSION THREAD

**Discussion by robylosa94** [idiotWu/smooth-scrollbar#349] [Q&A] [answered]
Can I change the scroll value?

**Comment by dolphin-wood** [comment, marked as answer]
https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarscrollleft

**Comment by robylosa94** [comment]
Thanks so much!

Il 31.01.2018 17:29 Dolphin Wood ha scritto: 

>
https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarscrollleft
[1] 
> 
> --
> You are receiving this because you authored the
thread.
> Reply to this email directly, view it on GitHub [2], or mute
the thread [3].
  


Con Mobile Open 7 GB a 9 euro/4 sett navighi veloce con 7 GB di Internet e hai 200 minuti ed SMS a 12 cent. Passa a Tiscali Mobile! http://tisca.li/OPEN7GBFirma
